src/it/scala/io/iohk/ethereum/rlp/RLPSpeedSuite.scala

```diff
@@ -1,8 +1,11 @@
 package io.iohk.ethereum.rlp
 
+import scala.language.implicitConversions
```

Comment:
I don't think we need this. Implicit parameters and implicit classes are not considered dangerous and do not require this import.